nixos/modules/services/networking/ntpd.nix

```diff
@@ -23,9 +23,11 @@ let
     restrict -6 ::1
 
     ${toString (map (server: "server " + server + " iburst\n") cfg.servers)}
+
+    ${cfg.extraConfig}
   '';
 
-  ntpFlags = "-c ${configFile} -u ${ntpUser}:nogroup ${toString cfg.extraFlags}";
+  ntpFlags = "-c ${configFile} -N -u ${ntpUser}:nogroup ${toString cfg.extraFlags}";
```

Comment:
Is this a good idea?

Reply:
Yes, if you care about a have a highly accurate system clock. A system under high load will have scheduling delays, which leads to increase jitter. I have never worked in a place that hasn't had -N.

Though, you are right that this can just be added to extraFlags, further reducing this diff.

Reply:
You don't need to minimize the diff. I think it's nice if NixOS has great, pleasantly surprising default configs. If you think 90% of people would want to use it with `-N` then please add it :)

I just wanted to make sure it's a good idea since the manual says it will result in ntpd being scheduled with a higher priority than the other processes - so that might affect system performance. (Which is probably of little disturbance with ntpd - but I wouldn't know).

If you think it should be the default but you should be able to turn it off  easily, add a config option for it and have it default to `true`.